### PR TITLE
Some mappings

### PR DESCRIPTION
**Pull request by ramidzkh**
Is it me or do 40% of the classes relate to collections?

### CODE REVIEW

File: mappings/fastcraft/compat/EnumForeignMods.mapping [hunk 1]
```diff
@@ -1,4 +1,6 @@
 CLASS fastcraft/k fastcraft/compat/EnumForeignMods
+	FIELD a values [Lfastcraft/k;
 	FIELD a entryPoints [Ljava/lang/String;
 	METHOD <init> (Ljava/lang/String;I[Ljava/lang/String;)V
 		ARG 3 entryPoints
+	METHOD values values ()[Lfastcraft/k;
```
Comment (Chocohead) [inline mappings/fastcraft/compat/EnumForeignMods.mapping:6]: Probably don't need this line

File: mappings/fastcraft/FastcraftClassTransformer.mapping [hunk 2]
```diff
@@ -0,0 +1,8 @@
+CLASS fastcraft/J fastcraft/FastcraftClassTransformer
```
Comment (valoeghese) [inline mappings/fastcraft/FastcraftClassTransformer.mapping:1]: `fastcraft/asm` package

File: mappings/fastcraft/FastcraftFMLTweaker.mapping [hunk 3]
```diff
@@ -0,0 +1,2 @@
+CLASS fastcraft/Tweaker fastcraft/FastcraftFMLTweaker
```
Comment (valoeghese) [inline mappings/fastcraft/FastcraftFMLTweaker.mapping:1]: isn't this already deobf?

File: mappings/fastcraft/MapHolder.mapping [hunk 4]
```diff
@@ -0,0 +1 @@
+CLASS fastcraft/aZ fastcraft/MapHolder
```
Comment (valoeghese) [inline mappings/fastcraft/MapHolder.mapping:1]: `fastcraft/util` package probably

File: mappings/fastcraft/ResizeUnicodeTextFontRenderer.mapping [hunk 5]
```diff
@@ -0,0 +1,2 @@
+CLASS fastcraft/U fastcraft/ResizeUnicodeTextFontRenderer
```
Comment (valoeghese) [inline mappings/fastcraft/ResizeUnicodeTextFontRenderer.mapping:1]: subpackage?

File: mappings/fastcraft/TextureAtlasSpriteComparator.mapping [hunk 6]
```diff
@@ -0,0 +1,4 @@
+CLASS fastcraft/aa fastcraft/TextureAtlasSpriteComparator
```
Comment (valoeghese) [inline mappings/fastcraft/TextureAtlasSpriteComparator.mapping:1]: subpackage?

File: mappings/fastcraft/TransformerHook.mapping [hunk 7]
```diff
@@ -0,0 +1 @@
+CLASS fastcraft/L fastcraft/TransformerHook
```
Comment (valoeghese) [inline mappings/fastcraft/TransformerHook.mapping:1]: `fastcraft/asm` package probably

File: mappings/fastcraft/ZipEntryInputStream.mapping [hunk 8]
```diff
@@ -0,0 +1,3 @@
+CLASS fastcraft/aE fastcraft/ZipEntryInputStream
```
Comment (valoeghese) [inline mappings/fastcraft/ZipEntryInputStream.mapping:1]: `fastcraft/util` package

**Review by valoeghese — CHANGES_REQUESTED** [review]

**Comment by valoeghese** [comment]
Hi resolve errors thank

**Comment by ramidzkh** [comment]
wtf is this

**Comment by valoeghese** [comment]
whatever I'll merge it

**Pull request merged**